lib/mktorrent.rb

```diff
@@ -152,6 +155,10 @@ def set_webseed(url)
     @webseed = url
   end
 
+  def add_tracker(tracker)
+    @tracker_list << Array.[](tracker)
```

Comment:
In the end announce_list should be like a [[tracker1], [tracker2], [tracker3]]
I tryed [[tracker1, tracker2]] , but it doesnt works for me